Fixups of README and other docs (#12146)
slint-ui/slint: FAQ.md

#### What counts as an embedded system?

> An **embedded system** is a computer system that performs a specific task within a larger mechanical or electrical system — for example the controller driving the screen of an appliance, a point-of-sale terminal, or a car dashboard. Using Slint this way needs the Commercial license.

Reviewer: ```suggestion
An **embedded system** is a computer system that performs a specific task within a larger mechanical or electrical system — for example the controller driving the screen of an appliance, a point-of-sale terminal, or a car dashboard. Select the Commercial license when building such a system with Slint.
```

or

```suggestion
An **embedded system** is a computer system that performs a specific task within a larger mechanical or electrical system — for example the controller driving the screen of an appliance, a point-of-sale terminal, or a car dashboard. The Commercial license or the GPL are suitable licenses for Slint for such systems.
```
